--- a/device.mk	2022-01-01 21:06:09.503000819 +0100
+++ b/device.mk	2022-01-01 22:23:35.949378909 +0100
@@ -22,20 +22,5 @@
 # System properties
 -include $(LOCAL_PATH)/system_prop.mk
 
-#GAPPS config
-GAPPS_VARIANT := pico
-GAPPS_PRODUCT_PACKAGES += Youtube
-GAPPS_PRODUCT_PACKAGES += GoogleDialer
-GAPPS_PRODUCT_PACKAGES += Wallpapers
-GAPPS_PRODUCT_PACKAGES += Chrome
-GAPPS_PRODUCT_PACKAGES += CalculatorGoogle 
-GAPPS_PRODUCT_PACKAGES += PrebuiltDeskClockGoogle
-GAPPS_PRODUCT_PACKAGES += CalendarGooglePrebuilt
-GAPPS_PRODUCT_PACKAGES += PrebuiltGmail
-GAPPS_PRODUCT_PACKAGES += GoogleContacts 
-GAPPS_PRODUCT_PACKAGES += Messaging 
-GAPPS_PRODUCT_PACKAGES += TrichromeLibraryGoogle
-$(call inherit-product, vendor/opengapps/build/opengapps-packages.mk)
-
 # Common overlay
 DEVICE_PACKAGE_OVERLAYS += $(LOCAL_PATH)/overlay
