--- a/vendor/motorola/exynos9610-common/exynos9610-common-vendor.mk	2022-04-05 23:09:21.611305800 +0200
+++ b/vendor/motorola/exynos9610-common/exynos9610-common-vendor.mk	2022-04-17 22:25:38.839985100 +0200
@@ -70,13 +70,8 @@
     vendor/motorola/exynos9610-common/proprietary/system_ext/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so:$(TARGET_COPY_OUT_SYSTEM_EXT)/lib64/vendor.samsung_slsi.telephony.hardware.radioExternal@1.0.so
 
 PRODUCT_PACKAGES += \
-    MotoSignatureApp \
     ShannonIms \
     OemRilService \
-    MotCamera2AI \
-    HotwordEnrollmentOKGoogleWM8280 \
-    HotwordEnrollmentXGoogleWM8280 \
-    MotCamera2 \
     ShannonDataService \
     ShannonNetworkService \
     ShannonQualifiedNetworksService
